account_promissory_note_bankia/report/check_print.py

```diff
@@ -0,0 +1,7 @@
+from odoo import models
+
+
+class ReportPromissoryNotePrintBankia(models.AbstractModel):
+    # Bankia
```

Comment:
I don't get why you need such abstract model. It's some kind of trick that is needed by account_check_printing_report_base for the report to work? If so, please put a comment about that.